tests/test_fetch.py

```diff
@@ -1,52 +0,0 @@
-from pathlib import Path
-
-import pytest
-
-from vrsix import fetch
-
-
-@pytest.fixture()
```

Comment:
this whole file would probably be good to add to a test on the vrs toolkit side, would you be comfortable adding this?

Reply:
Yeah, definitely. Sounds good to me.